bounties/level-2/million-lumen-homepage.md

```diff
@@ -0,0 +1,34 @@
+
+# Million Lumen Homepage
+
+| Hunter | Agent
+| :- | :-
+| 800 XLM | 2 Credits
+
+## Description
+
+### What is this task?
+
+Remember the Million Dollar Homepage? It was an interesting concept, and it's high time we implement it on Stellar!
+The task is to create a grid of million pixels up for sale and the cost of each individual pixel is 1 XLM.
+
+### What are the requirements for the bounty hunter?
+
+The developer should have knowledge of Web Technologies and Stellar SDK.
+
+### What are the deliverables?
+
+*Description* <br>
+  * The developer should provide a link to a repository implementing the new functionality.
+
+*Judging Criteria and Metrics* <br>
+  * MVP - Does the functionality match with what's stated in the proposal? And how similar it is to the original concept?
```

Comment:
> stated in the proposal

What proposal? I assume maybe the wikipedia page? The proposal should probably be directly linked to here.